resources/assets/lib/chat/message-group.tsx

```diff
@@ -44,46 +41,20 @@ export default class MessageGroup extends React.Component<Props, any> {
         </div>
         <div className='chat-message-group__bubble'>
           {messages.map((message: Message, key: number) => {
-            if (!message.parsedContent) {
+            if (message.parsedContent == null) {
               return;
             }
 
-            let classes = 'chat-message-group__message';
-            let contentClasses = 'chat-message-group__message-content';
-
-            if (!message.persisted) {
-              classes += ' chat-message-group__message--sending';
-            }
-
-            if (message.isAction) {
-              contentClasses += ' chat-message-group__message-content--action';
-            }
-
             const showTimestamp: boolean =
               // show timestamp if this is the last message in the group
               (key === messages.length - 1) ||
               // or if the next message has a different displayed timestamp
               (moment(message.timestamp).format('LT') !== moment(messages[key + 1].timestamp).format('LT'));
 
             return (
-              <div key={message.uuid} className={classes}>
-                <div className='chat-message-group__message-entry'>
-                  <span className={contentClasses} dangerouslySetInnerHTML={{__html: message.parsedContent}} />
-                  {!message.persisted && !message.errored &&
-                    <div className='chat-message-group__message-status'>
-                      <Spinner />
-                    </div>
-                  }
-                  {message.errored &&
-                    <div className='chat-message-group__message-status chat-message-group__message-status--errored'>
-                      <i className='fas fa-times'/>
-                    </div>
-                  }
-                </div>
-                { showTimestamp &&
-                  <div className='chat-message-group__message-timestamp'>{moment(message.timestamp).format('LT')}</div>
-                }
-              </div>
+              <MessageErrorBoundary key={message.uuid}>
+                <MessageItem message={message} showTimestamp={showTimestamp} />
```

Comment:
`MessageItem` doesn't throw anything and it doesn't contain anything that's obviously going to throw on error so I don't think the boundary is useful here?

Not to mention the component is also sometimes responsible rendering timestamp (though this part could be separated by pre-grouping the messages by timestamp and always render timestamp separate at the end).